tests/test_sync_without_refrence_organization_id.py

```diff
@@ -23,9 +23,9 @@ def test_run(self):
                                       if catalog.get('stream_name') in expected_streams]
 
         self.perform_and_verify_table_and_field_selection(conn_id, test_catalogs, non_selected_properties=['reference_organization_id'])
-        
+
         record_count_by_stream = self.run_and_verify_sync(conn_id)
 
         # check if all streams have collected records
         for stream in expected_streams:
-            self.assertGreater(record_count_by_stream.get(stream, 0), 0)
\ No newline at end of file
+            self.assertGreater(record_count_by_stream.get(stream, 0), 0)
```

Comment:
Please update clear objective of this test in doc string and why we are handling it separately since this is not a standard test.

Reply:
There is a field `reference_organization_id` in the records of the `users` stream. If this field is absent, then the tap was throwing an error. It is fixed now and if this field is missing then the[ `warning`](https://github.com/singer-io/tap-linkedin-ads/blob/4400a536329e10f3af831811b184d4e6ff62e37c/tap_linkedin_ads/streams.py#L379-#L384) message would print. This test case is written to verify that case. But, this scenario is already covered in the [unit test case](https://github.com/singer-io/tap-linkedin-ads/blob/TDL-20154-dict-base-to-class-base-implementation/tests/unittests/test_streams.py#L376). So, removed this tap tester test case.